templates/index.html

```diff
@@ -33,6 +33,35 @@
   </header>
 
   <div id="builds">
+
+    <!-- B2GDroid promo -->
+    <!--  Spotlight Activity Tag: Mozilla (6247) | Mozilla B2G Activity Tag 2 (52162) | Mozilla Spots (4669) | Expected URL:  -->
+    <script>
+      function ftGoalTag52162() {
+        var ftRand = Math.random() + "";
+        var num = ftRand * 1000000000000000000;
+        var ftGoalTagPix52162 = new Image();
+        ftGoalTagPix52162.src = "http://servedby.flashtalking.com/spot/8/6247;52162;4669/?spotName=Mozilla_B2G_Activity_Tag_2&cachebuster="+num;
+        ftGoalTagPix52162.onload = ftLoaded52162;
+      }
+      function ftLoaded52162() {
+        top.location.href="https://d2yw7jilxa8093.cloudfront.net/B2GDroid-mozilla-central-nightly-latest.apk";
+      }
+    </script>
+    <div id="Firefox OS" class="build-group">
```

Comment:
To add a new build group/item please add it to the `scrape.py` file: https://github.com/mozilla/nocturnal/blob/master/scrape.py#L43

Your proposed solution won't update the file names when new versions are released, plus it's easier to keep track of all build items in the same place (`scrape.py`).